plugins/Model_GAN128/Trainer.py

```diff
@@ -74,6 +75,12 @@ def setup(self):
         loss_GA += 3e-3 * K.mean(K.abs(mask_A))
         loss_GB += 3e-3 * K.mean(K.abs(mask_B))
 
+        w_fo = 0.01
+        loss_GA += w_fo * K.mean(first_order(mask_A, axis=1))
+        loss_GA += w_fo * K.mean(first_order(mask_A, axis=2))
+        loss_GB += w_fo * K.mean(first_order(mask_B, axis=1))
+        loss_GB += w_fo * K.mean(first_order(mask_B, axis=2))

```

Comment:
This is the main difference between Clorr's current gan128 implementation and shaoanlu's latest gan128. shaoanlu added it in the commit https://github.com/shaoanlu/faceswap-GAN/commit/ea6b27b5e665748285de34a52aed767459740674